src/explorer/resultsList.ts

```diff
@@ -748,4 +732,62 @@ export class ResultsList {
             caseButton.classList.remove("active");
         }
     }
+
+    /**
+     * Handler to perform keyboard navigation in the results list.
+     * @param keydownEvent Keyboard event
+     */
+    private onResultsListKeyDown(keydownEvent: JQuery.KeyDownEvent): void  {        
+        enum KeyCodes {
+            LeftArrow = 37,
+            UpArrow = 38,
+            RightArrow = 39,
+            DownArrow = 40
+        }
+
+        let direction: 'Previous' | 'Next' | undefined;
+        if (keydownEvent.keyCode === KeyCodes.LeftArrow || keydownEvent.keyCode === KeyCodes.UpArrow) {
```

Comment:
`.keyCode` is deprecated. `.key` is preferred.

Reply:
Yep. Just noticed that... Looking into this

---
In reply to: [413140401](https://github.com/microsoft/sarif-vscode-extension/pull/254#discussion_r413140401) [](ancestors = 413140401)